assertj-core/src/main/java/org/assertj/core/util/introspection/CaseStyleInsensitiveFieldLookupStrategy.java

```diff
@@ -0,0 +1,36 @@
+package org.assertj.core.util.introspection;
+
+import static java.util.stream.Collectors.joining;
+import static java.util.stream.Collectors.toList;
+import static org.assertj.core.api.recursive.comparison.ComparingSnakeOrCamelCaseFields.COMPARING_SNAKE_OR_CAMEL_CASE_FIELDS;
```

Comment:
a `util` package should not depend on the recursive comparison package, should be the other way around.